pkg/arvo/app/s3-store.hoon

```diff
@@ -89,7 +89,17 @@
   --
 ::
 ++  on-leave  on-leave:def
-++  on-peek   on-peek:def
+++  on-peek
+  ~/  %s3-peek
+  |=  =path
+  ^-  (unit (unit cage))
+  ?>  (team:title our.bowl src.bowl)
+  ?+  path      (on-watch:def path)
+      [%x %credentials ~]
```

Comment:
- Usually queenside form has the opposite indentation to here i.e.
```hoon
?+    path  (on-watch:def)
    [%x credentials]
  [~ ~ %s3-update !>(`noun`[%credentials credentials])]
```
etc. 

- The cast to `noun` will make scrying these paths more expensive. If the cast is changed to `update`, then most instances of scry typechecking will be trivial because de-vasing with the same type of the original `!>` is a no-op.

Reply:
@liam-fitzgerald please review, again. I've made (i think) the requested changes.